### PR TITLE
Add autodeployment changes

### PR DESCRIPTION
**Pull request by mamedin**
This PR will:

* Update some CLI commands
* Add the new CLI command to configure and register pipeline
* Add new API section to the security document

These changes come from https://github.com/artefactual/archivematica/pull/681 , that includes this new CLI command and a major change in the API whitelist.

### CODE REVIEW

**Comment by sromkey** [comment]
@scollazo Would you be able to review this PR when you have a chance? Thanks!

File: admin-manual/security/security.rst [hunk 1]
```diff
@@ -12,6 +12,22 @@ Security
...
+API access control
+------------------
+
+The mechanism chosen to restrict access to the API is the combination of an API
+key and a whitelist where the allowed hosts are defined.
+
+By restricting the IP addresses the impact of a compromised API key can be
+reduced.
+
+There has been a major change in AM version 1.7 on the whitelist definition. In
```
Comment (scollazo) [inline admin-manual/security/security.rst]: Should we put this in bold? Or other format to draw attention to it

**Review by scollazo — CHANGES_REQUESTED** [review]

**Comment by mamedin** [comment]
Hi Santi, I have set as "important" the paragraph you said. Thanks for the suggestion :)

File: admin-manual/installation/dashboard-config.rst [hunk 2]
```diff
@@ -557,10 +557,14 @@ command-line, issue the following commands:
 
 .. code:: bash
 
+   sudo su - archivematica -s /bin/bash
+   export $(grep -v ^# /etc/default/archivematica-dashboard)
    cd /usr/share/archivematica/dashboard
-   export PATH=$PATH:/usr/share/archivematica/dashboard
-   export DJANGO_SETTINGS_MODULE=settings.common
-   python manage.py createsuperuser
+   /usr/share/python/archivematica-dashboard/bin/python manage.py createsuperuser
```
Comment (scollazo) [inline admin-manual/installation/dashboard-config.rst]: We should list the paramaters as we do for the dashboard manage.py command

**Review by scollazo — CHANGES_REQUESTED** [review]

**Comment by sallain** [comment]
@mamedin Is this work going to be picked up? Or have these changes been reflected in other PRs since?

**Comment by mamedin** [comment]
Hi, I updated this PR to AM 1.8.

Initially this PR had 3 commits, but now it is only needed this commit. It is ready for CR again.

File: admin-manual/installation-setup/customization/dashboard-config.rst [hunk 3]
```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
        /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python manage.py changepassword <username>
    ";
 
+CLI configuration pipeline and registration on the Storage Service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On new installations, the superuser can be configured and the pipeline
+registered on the Storage Service using the command-line instead of the GUI.
+These commands configure the pipeline and register it on the storage service
```
Comment (sallain) [inline admin-manual/installation-setup/customization/dashboard-config.rst]: s/storage service/Storage Service

File: admin-manual/installation-setup/customization/dashboard-config.rst [hunk 4]
```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
...
+exactly like the GUI for the initial configuration screen shown after the
+installation would:
+
```
Comment (sallain) [inline admin-manual/installation-setup/customization/dashboard-config.rst:770]: I think you can remove "shown after the installation would" for clarity.

File: admin-manual/installation-setup/customization/dashboard-config.rst [hunk 5]
```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
...
+      the Storage Service for the user <ss-username>.
+
+    * whitelist: Whitespace-separated list of IP addresses or hostnames allowed
+      to use the API. If the whitelist is left empty, all IP addresses and
+      hostnames will be allowed. For example: "192.168.1.3
+      example.archivematica.org 127.0.0.1" or "".
+
+This command is most suitable to use on automated installations, for example
+when deploying using ansible. For manual installations, please use the web
+configuration method.
```
Comment (sallain) [inline admin-manual/installation-setup/customization/dashboard-config.rst]: Provide a link to the web config method if possible.

File: admin-manual/installation-setup/customization/dashboard-config.rst [hunk 6]
```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
...
+           --ss-url=<ss-ulr> \
+           --ss-user=<ss-username> \
+           --ss-api-key=<ss-api-key> \
+           --whitelist=<whitelist>
+   ";
+
+Where:
+
+    * api-key: Api key that will be added to the username <username>.
+      For example: "test", "4e5f32ab2aefd3577e1b19a2de5d4dd65f90101a".
```
Comment (sallain) [inline admin-manual/installation-setup/customization/dashboard-config.rst]: s/Api key/API key

**Review by sallain — CHANGES_REQUESTED** [review]

File: admin-manual/installation-setup/customization/dashboard-config.rst [hunk 7]
```diff
@@ -721,6 +721,59 @@ user, you can change it via the command-line:
...
+           --org-name=<org-name> \
+           --org-id=<org-id> \
+           --api-key=<api-key>\
+           --ss-url=<ss-ulr> \
+           --ss-user=<ss-username> \
+           --ss-api-key=<ss-api-key> \
+           --whitelist=<whitelist>
+   ";
+
+Where:
```
Comment (sevein) [inline admin-manual/installation-setup/customization/dashboard-config.rst:792]: In the list below, can you use [inline literals](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup) for things like `api-key`, `test` etc?

Also the indentation in the enumeration is not needed, the fact that is there causes the HTML to be rendered inside a `blockquote`, see:

image

**Comment by mamedin** [comment]
Thanks Sara and Jesus for your suggestions :). I added them and it is ready for CR again.

**Review by sevein — APPROVED** [review]
Thanks!

**Review by sallain — APPROVED** [review]

**Pull request merged**